tool/jt.rb

```diff
@@ -679,12 +694,7 @@ def run_ruby(*args)
       options[:use_exec] = true
     end
 
-    ruby_bin = if args.delete('--aot')
-                 verify_aot_bin!
-                 ENV['AOT_BIN']
-               else
-                 Utilities.find_launcher
-               end
+    ruby_bin = Utilities.find_launcher(args.delete('--native'))
```

Comment:
Should there be a `verify_aot_bin!` call in `find_launcher` if passed native=true?

Reply:
It doesn't do any harm, so I can add it. The reason I have the others outside of this call is we use this in subprocesses for things like benchmarks and metrics and I didn't want the verification message to get swallowed. So I pushed the verification up and earlier in the call sequence.

Reply:
Note that `verify_aot_bin!` also relies on `find_launcher` returning `nil` if a launcher can't be found. I'd have to restructure a bit.